### PR TITLE
Get File URL and contentType of a file path from Firebase Storage Bucket

### PR DESCRIPTION
**Pull request by hugomsilvam**
Add new features to access the file URL from Firebase Storage Bucket given a path (example challenge/{id}/media).

Returns the URL (string) and the content type (📹 📷 ) of a file.

### CODE REVIEW

**Review by ruimartinsptl — APPROVED** [review]
I don't have anything to comment

File: src/actions/storage.js [hunk 1]
```diff
@@ -16,6 +20,73 @@ const {
...
+      dispatch({
+        type: FILE_GET_DOWNLOAD_URL_START,
+        payload
+      })
+      return payload
+    })
+    .catch(error => {
+      /* eslint-disable no-console */
+      console.error('failed to get file path ', path, ' with error ', error)
+      /* eslint-disable no-console */
```
Comment (portellaa) [inline src/actions/storage.js]: shouldn't this be `eslint-enable` 🤔

File: src/actions/storage.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ import {
 } from '../utils/storage'
 
 const {
+  FILE_GET_DOWNLOAD_URL_START,
+  FILE_GET_DOWNLOAD_URL_ERROR,
```
Comment (portellaa) [inline src/actions/storage.js:12]: shouldn't you have a `FILE_GET_DOWNLOAD_URL_COMPLETE`?

Comment (portellaa) [reply src/actions/storage.js:12]: or perhaps remove the start and just have `FILE_GET_DOWNLOAD_URL`

File: src/createFirebaseInstance.js [hunk 3]
```diff
@@ -233,6 +233,45 @@ export const createFirebaseInstance = (firebase, configs, dispatch) => {
...
+  /**
+   * @description get file content type from Firebase Storage
+   * @param {String} path - File path in the Firebase Storage Bucket
+   * @returns {String} File ContentType
+   */
+  const getFileContentType = path => {
+    return storageActions
+      .getMetadata(dispatch, firebase, { path })
+      .then(({ metadata }) => {
+        return metadata.contentType
```
Comment (portellaa) [inline src/createFirebaseInstance.js]: 💄  you could remove the `{ }` and the `return`

File: src/createFirebaseInstance.js [hunk 4]
```diff
@@ -233,6 +233,45 @@ export const createFirebaseInstance = (firebase, configs, dispatch) => {
...
+  }
+
+  /**
+   * @description get file url and content type from Firebase Storage
+   * @param {String} path - FIle path in the Firebase Storage Bucket
+   * @returns {Object} result - File URL {String} and ContentType {String}
+   */
+  const getFileURLAndContentType = path => {
+    return Promise.all([getFileURL(path), getFileContentType(path)]).then(
+      result => {
```
Comment (portellaa) [inline src/createFirebaseInstance.js]: 💄 same as above

**Review by portellaa — APPROVED** [review]
finished my review with small suggestions. nice work

**Comment by portellaa** [comment]
Don't forget to use Squash and merge

**Pull request merged**